src/server/scripts/Events/hallows_end.cpp

```diff
@@ -57,13 +83,452 @@ class spell_hallows_end_has_water_bucket : public SpellScriptLoader
         }
     };
 
-    AuraScript* GetAuraScript() const
+    AuraScript* GetAuraScript() const override
     {
         return new spell_hallows_end_has_water_bucket_AuraScript();
     }
 };
 
+class spell_hallows_end_bucket_lands : public SpellScriptLoader
+{
+public:
+    spell_hallows_end_bucket_lands() : SpellScriptLoader("spell_hallows_end_bucket_lands") {}
+
+    class spell_hallows_end_bucket_lands_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_hallows_end_bucket_lands_SpellScript);
+
+        bool handled;
+        bool Load() { handled = false; return true; }
```

Comment:
[ 39%] /home/travis/build/TrinityCore/TrinityCore/src/server/scripts/Events/hallows_end.cpp:68:14: fatal error: 
      'Load' overrides a member function but is not marked 'override'
      [-Winconsistent-missing-override]
        bool Load() { handled = false; return true; }
             ^
/home/travis/build/TrinityCore/TrinityCore/src/server/game/Spells/SpellScript.h:131:22: note: 
      overridden virtual function is here
        virtual bool Load() { return true; }
                     ^